core/src/main/resources/hudson/tasks/ArtifactArchiver/help-flattenDirectories.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  Optionally flatten directories to be archived, so that all archived files are
+    placed in the same archive directory. Note that this will fail builds if
+    there are files with identical filenames to be archived.
```

Comment:
Could this be improved by writing *multiple files with the same name* or something similar?